pkg/compactor/blocks_cleaner_test.go

```diff
@@ -743,6 +839,13 @@ func (m *mockConfigProvider) CompactorBlockUploadEnabled(tenantID string) bool {
 	return m.blockUploadEnabled[tenantID]
 }
 
+func (m *mockConfigProvider) CompactorPartialBlockDeletionDelay(user string) time.Duration {
+	if result, ok := m.userPartialBlockDelay[user]; ok {
+		return result
+	}
+	return 0
```

Comment:
Nit: This will work too:

```suggestion
	return m.userPartialBlockDelay[user]
```

because if map doesn't have anything for the user, default value (which is 0) is returned.

Reply:
Agree with @pstibrany, I noticed the same :)